### PR TITLE
PCX-957: Add network logos to grouped card

### PR DESCRIPTION
**Pull request by siebeprojects**

### CODE REVIEW

File: payment/src/main/java/net/optile/payment/ui/model/SmartSwitch.java [hunk 1]
```diff
@@ -66,6 +66,15 @@ public boolean hasSelected() {
         return smartSelected.size() > 0;
     }
 
+    /**
+     * Get the number of selected networks
+     *
+     * @return the number of selected networks
+     */
+    public int getSelectedSize() {
```
Comment (hjhill) [inline payment/src/main/java/net/optile/payment/ui/model/SmartSwitch.java:74]: `Size` reads a bit strange in that context - better name it `getSelectedCount`

File: payment/src/main/java/net/optile/payment/ui/list/NetworkCardViewHolder.java [hunk 2]
```diff
@@ -59,16 +62,37 @@ void onBind(PaymentCard paymentCard) {
...
+        if (smartSwitch.getSelectedSize() == 1) {
+            PaymentNetwork network = smartSwitch.getFirstSelected();
+            networkLogosView.setSelected(network.getCode());
+            return;
+        }
+        networkLogosView.setSelected(null);
+    }
+
+    private void addNetworkLogos(View parent, NetworkCard networkCard) {
+        if (networkCard.getPaymentNetworkSize() <= 1) {
```
Comment (hjhill) [inline payment/src/main/java/net/optile/payment/ui/list/NetworkCardViewHolder.java:90]: This is another case of `Size` where `Count` would probably be better - or

File: payment/src/main/java/net/optile/payment/ui/model/SmartSwitch.java (hunk 1 above)
Comment (siebeprojects) [reply payment/src/main/java/net/optile/payment/ui/model/SmartSwitch.java:74]: Changed to count

File: payment/src/main/java/net/optile/payment/ui/list/NetworkCardViewHolder.java (hunk 2 above)
Comment (siebeprojects) [reply payment/src/main/java/net/optile/payment/ui/list/NetworkCardViewHolder.java]: Changed as well to count

**Pull request merged**